docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,248 @@
+#####################################################
+Compute circuit output probabilities with ``Sampler``
+#####################################################
+
+This guide shows how to get the probability distribution of a quantum circuit with the :class:`~qiskit.primitives.Sampler` primitive.
+
+.. note::
+
+    While this guide uses Qiskit’s reference implementation, the ``Sampler`` primitive can be run with any provider using :class:`~qiskit.primitives.BackendSampler`.
+    
+    .. code-block::
+
+        from qiskit.primitives import BackendSampler
+        from qiskit_provider import QiskitProvider
+
+        provider = QiskitProvider()
+        backend = provider.get_backend('backend_name')
+        sampler = BackendSampler(backend)
+
+    There are some providers that implement primitives natively (see `this page <http://qiskit.org/providers/#primitives>`_ for more details).
+
+Initialize quantum circuits
+===========================
+
+The first step is to create the :class:`~qiskit.circuit.QuantumCircuit` from which you want to obtain the probability distribution.
+
+.. plot::
+    :include-source:
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0,1)
+    qc.measure_all()
+    qc.draw("mpl")
+
+.. testsetup::
```

Comment:
ditto https://github.com/Qiskit/qiskit-terra/pull/9716/files#r1153023515 ?

Reply:
Fixed in [36d54f3](https://github.com/Qiskit/qiskit-terra/pull/9716/commits/36d54f3a625d12fbec8c9c03c4cc93005ff22be2)